docs/blog/oras-0.14-and-future.md

```diff
@@ -107,7 +107,7 @@ ORAS 0.15 and future milestones will provide more capabilities to easily manage
 - Support push/pull artifacts from OCI Image Layout
 - E2E testing
 
-See the ORAS [Roadmap](https://github.com/oras-project/community/blob/main/Roadmap.md) for more details.
+See the ORAS [Roadmap](https://github.com/oras-project/oras/projects/1) for more details.
```

Comment:
Thank you @bridgetkromhout for contributing to the ORAS website!

Now the [Roadmap link](https://github.com/oras-project/community/blob/main/Roadmap.md) is available, could you withdraw this update? Others are good to me.